src/tags/include_tag.rs

```diff
@@ -0,0 +1,76 @@
+
+use Renderable;
+use context::Context;
+use token::Token;
+use LiquidOptions;
+use template::Template;
+use parser;
+use lexer;
+use error::{Result, Error};
+
+use std::fs::File;
+use std::io::Read;
+use std::path::Path;
+
+struct Include {
+    partial: Template,
+}
+
+impl Renderable for Include {
+    fn render(&self, mut context: &mut Context) -> Result<Option<String>> {
+        self.partial.render(&mut context)
+    }
+}
+
+fn parse_partial(path: &str, options: &LiquidOptions) -> Result<Template> {
+    let path = Path::new(&path);
+
+    // check if file exists
+    if !path.exists() {
+        return Error::parser_msg(&format!("{:?} does not exist", path));
+    }
```

Comment:
@johannhof You are right but the reason I did not use that error is because the error is just `Io(Error { repr: Os { code: 2, message: "No such file or directory" } })'` where the error message that I use has the file name in it. `Parser("\"file_does_not_exsist.liquid\" does not exist")`. To me this seems like the better option that will notify the user that the included file is not a file without having to run the command with `--trace` to see which file could not be found.